conductor-client/src/main/kotlin/com/openlattice/collaborations/PostgresCollaborationDatabaseService.kt

```diff
@@ -0,0 +1,256 @@
+package com.openlattice.collaborations
+
+import com.hazelcast.core.HazelcastInstance
+import com.hazelcast.query.Predicate
+import com.hazelcast.query.Predicates
+import com.openlattice.assembler.Assembler
+import com.openlattice.assembler.AssemblerConfiguration
+import com.openlattice.assembler.AssemblerConnectionManager
+import com.openlattice.authorization.AclKey
+import com.openlattice.authorization.AuthorizationManager
+import com.openlattice.authorization.DbCredentialService
+import com.openlattice.authorization.Permission
+import com.openlattice.collaborations.mapstores.ProjectedTablesMapstore
+import com.openlattice.hazelcast.HazelcastMap
+import com.openlattice.organizations.OrganizationDatabase
+import com.openlattice.organizations.mapstores.TABLE_ID_INDEX
+import com.openlattice.organizations.roles.SecurePrincipalsManager
+import com.openlattice.postgres.PostgresProjectionService
+import com.openlattice.postgres.external.ExternalDatabaseConnectionManager
+import com.openlattice.postgres.external.ExternalDatabasePermissioningService
+import com.openlattice.postgres.external.Schemas
+import com.zaxxer.hikari.HikariDataSource
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class PostgresCollaborationDatabaseService(
+        hazelcast: HazelcastInstance,
+        val hds: HikariDataSource,
+        val assembler: Assembler,
+        val acm: AssemblerConnectionManager,
+        val externalDbConnMan: ExternalDatabaseConnectionManager,
+        val authorizations: AuthorizationManager,
+        val externalDbPermissioner: ExternalDatabasePermissioningService,
+        val spm: SecurePrincipalsManager,
+        val dbCreds: DbCredentialService,
+        val assemblerConfiguration: AssemblerConfiguration
+) : CollaborationDatabaseManager {
```

Comment:
Just a follow-up to the earlier comment in ACM, if you move that stuff out of ACM, you'll only need to depend on ExternalDatabaseManagementService in here instead of both ACM and Assembler 🔥